string robotName
string node_num
---
bool success
